apps/vr-tests-react-components/package.json

```diff
@@ -33,6 +33,7 @@
     "@fluentui/react-datepicker-compat": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29709&commitId=e582a0eac498f66ef943e90978dbd0fc27df0f5b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Drawer</b> 76 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.default drawer - Dark Mode.chromium.png|0| Added|
|Drawer.default drawer - High Contrast.chromium.png|0| Added|
|Drawer.default drawer - RTL.chromium.png|0| Added|
|Drawer.default drawer.chromium.png|0| Added|
|Drawer.inline drawer - Dark Mode.chromium.png|0| Added|
|Drawer.inline drawer - High Contrast.chromium.png|0| Added|
|Drawer.inline drawer - RTL.chromium.png|0| Added|
|Drawer.inline drawer full - Dark Mode.chromium.png|0| Added|
|Drawer.inline drawer full - High Contrast.chromium.png|0| Added|
|Drawer.inline drawer full - RTL.chromium.png|0| Added|
|Drawer.inline drawer full.chromium.png|0| Added|
|Drawer.inline drawer large - Dark Mode.chromium.png|0| Added|
|Drawer.inline drawer large - Dark Mode.chromium_1.png|0| Added|
|Drawer.inline drawer large - High Contrast.chromium.png|0| Added|
|Drawer.inline drawer large - High Contrast.chromium_1.png|0| Added|
|Drawer.inline drawer large - RTL.chromium.png|0| Added|
|Drawer.inline drawer large - RTL.chromium_1.png|0| Added|
|Drawer.inline drawer large content scroll - Dark Mode.chromium.png|0| Added|
|Drawer.inline drawer large content scroll - High Contrast.chromium.png|0| Added|
|Drawer.inline drawer large content scroll - RTL.chromium.png|0| Added|
|Drawer.inline drawer large content scroll.chromium.png|0| Added|
|Drawer.inline drawer large.chromium.png|0| Added|
|Drawer.inline drawer large.chromium_1.png|0| Added|
|Drawer.inline drawer position end - Dark Mode.chromium.png|0| Added|
|Drawer.inline drawer position end - High Contrast.chromium.png|0| Added|
|Drawer.inline drawer position end - RTL.chromium.png|0| Added|
|Drawer.inline drawer position end.chromium.png|0| Added|
|Drawer.inline drawer position start - Dark Mode.chromium.png|0| Added|
|Drawer.inline drawer position start - High Contrast.chromium.png|0| Added|
|Drawer.inline drawer position start - RTL.chromium.png|0| Added|
|Drawer.inline drawer position start.chromium.png|0| Added|
|Drawer.inline drawer separator position end - Dark Mode.chromium.png|0| Added|
|Drawer.inline drawer separator position end - High Contrast.chromium.png|0| Added|
|Drawer.inline drawer separator position end - RTL.chromium.png|0| Added|
|Drawer.inline drawer separator position end.chromium.png|0| Added|
|Drawer.inline drawer separator position start - Dark Mode.chromium.png|0| Added|
|Drawer.inline drawer separator position start - High Contrast.chromium.png|0| Added|
|Drawer.inline drawer separator position start - RTL.chromium.png|0| Added|
|Drawer.inline drawer separator position start.chromium.png|0| Added|
|Drawer.inline drawer.chromium.png|0| Added|
|Drawer.overlay drawer - Dark Mode.chromium.png|0| Added|
|Drawer.overlay drawer - High Contrast.chromium.png|0| Added|
|Drawer.overlay drawer - RTL.chromium.png|0| Added|
|Drawer.overlay drawer alert - Dark Mode.chromium.png|0| Added|
|Drawer.overlay drawer alert - High Contrast.chromium.png|0| Added|
|Drawer.overlay drawer alert - RTL.chromium.png|0| Added|
|Drawer.overlay drawer alert.chromium.png|0| Added|
|Drawer.overlay drawer full - Dark Mode.chromium.png|0| Added|
|Drawer.overlay drawer full - High Contrast.chromium.png|0| Added|
|Drawer.overlay drawer full - RTL.chromium.png|0| Added|
|Drawer.overlay drawer full.chromium.png|0| Added|
|Drawer.overlay drawer large - Dark Mode.chromium.png|0| Added|
|Drawer.overlay drawer large - High Contrast.chromium.png|0| Added|
|Drawer.overlay drawer large - RTL.chromium.png|0| Added|
|Drawer.overlay drawer large content scroll - Dark Mode.chromium.png|0| Added|
|Drawer.overlay drawer large content scroll - High Contrast.chromium.png|0| Added|
|Drawer.overlay drawer large content scroll - RTL.chromium.png|0| Added|
|Drawer.overlay drawer large content scroll.chromium.png|0| Added|
|Drawer.overlay drawer large.chromium.png|0| Added|
|Drawer.overlay drawer medium - Dark Mode.chromium.png|0| Added|
|Drawer.overlay drawer medium - High Contrast.chromium.png|0| Added|
|Drawer.overlay drawer medium - RTL.chromium.png|0| Added|
|Drawer.overlay drawer medium.chromium.png|0| Added|
|Drawer.overlay drawer non modal - Dark Mode.chromium.png|0| Added|
|Drawer.overlay drawer non modal - High Contrast.chromium.png|0| Added|
|Drawer.overlay drawer non modal - RTL.chromium.png|0| Added|
|Drawer.overlay drawer non modal.chromium.png|0| Added|
|Drawer.overlay drawer position end - Dark Mode.chromium.png|0| Added|
|Drawer.overlay drawer position end - High Contrast.chromium.png|0| Added|
|Drawer.overlay drawer position end - RTL.chromium.png|0| Added|
|Drawer.overlay drawer position end.chromium.png|0| Added|
|Drawer.overlay drawer position start - Dark Mode.chromium.png|0| Added|
|Drawer.overlay drawer position start - High Contrast.chromium.png|0| Added|
|Drawer.overlay drawer position start - RTL.chromium.png|0| Added|
|Drawer.overlay drawer position start.chromium.png|0| Added|
|Drawer.overlay drawer.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>